backend/src/main/java/site/coduo/referencelink/domain/OpenGraph.java

```diff
@@ -0,0 +1,59 @@
+package site.coduo.referencelink.domain;
+
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class OpenGraph {
+
+    private static final String EMPTY = "";
+    private static final String OPEN_GRAPH_META_TAG_SELECTOR = "meta[property=og:%s]";
+
+    private final String headTitle;
+    private final String openGraphTitle;
+    private final String description;
+    private final String image;
+
+    public OpenGraph() {
+        headTitle = EMPTY;
+        this.openGraphTitle = EMPTY;
+        this.description = EMPTY;
+        this.image = EMPTY;
+    }
+
+    @Builder
+    public OpenGraph(final String headTitle,
+                     final String openGraphTitle,
+                     final String description,
+                     final String image
+    ) {
+        this.headTitle = headTitle;
+        this.openGraphTitle = openGraphTitle;
+        this.description = description;
+        this.image = image;
+    }
```

Comment:
[컨벤션]
builder 사용시 생성자 접근제한자 private으로 막아놓기로 통일했던것으로 기억해요.
```suggestion
    @Builder
    private OpenGraph(final String headTitle,
                     final String openGraphTitle,
                     final String description,
                     final String image
    ) {
        this.headTitle = headTitle;
        this.openGraphTitle = openGraphTitle;
        this.description = description;
        this.image = image;
    }
```